composables/use-issues-by-column.ts

```diff
@@ -0,0 +1,40 @@
+import type { ColumnTitle } from '~/constants/columns'
+import type { IssueStatus } from '~/constants/issues'
+import type { Issue } from '~/types/issues'
+
+function filterIssuesWithoutStatus(issues: Issue[]): Issue[] {
+  const issuesWithoutStatus = issues.filter(
+    (issue) =>
+      !issue.labels.some((label) => label.startsWith(createPartialDataLabel('status'))),
+  )
+
+  return issuesWithoutStatus
+}
+
+function filterIssuesByStatus(issues: Issue[], status: IssueStatus): Issue[] {
+  const issuesWithStatus = issues.filter((issue) =>
+    issue.labels.includes(createDataLabel('status', status)),
+  )
+
+  return issuesWithStatus
+}
+
+function useIssuesByColumn(): Record<ColumnTitle, ComputedRef<Issue[]>> {
+  const route = useRoute('node-rid')
+  const response = useHttpdFetch('/projects/{rid}/issues', {
+    path: {
+      rid: route.params.rid,
+    },
+  })
+
+  const issues = response.data as Ref<Issue[] | undefined>
+
+  return {
```

Comment:
Never return non-trivial expressions. Best to always return a variable even if the expression resolving it is simple. A codebase with this practice is a pain in the a$$ to debug (which is when you are already in pain by definition). One has to refactor every time before being able to console.log() or add a debug statement and flamegraphs in perf audits are harder to read too.

Classic case of a bad practice learned due to react/JSX coding.

Reply:
Good to know! 

Assigned to a variable before returning